src/WinWrapper.cpp

```diff
@@ -303,6 +303,9 @@ int main(int argc, char** argv)
             gSettings.lockFrameRate = true;
             gSettings.lockedFrameRate = atoi(argv[++a]);
             printf("FPS locked to %u\n", gSettings.lockedFrameRate);
+        } else if (_stricmp(argv[a], "-num_asteroids") == 0 && a + 1 < argc) {
+            gSettings.numAsteroids = atoi(argv[++a]);
+            printf("FPS locked to %u\n", gSettings.lockedFrameRate);
```

Comment:
Copy-paste error here: should print something about num asteroids being overridden.  Also, missing a printf in the usage below.